include/tcp_ip.h

```diff
@@ -0,0 +1,34 @@
+#ifndef __TCP_IP_H__
+#define __TCP_IP_H__
+
+// Include custom debug macros and compile flags
+#include "flags.h"
+
+#if WIFI_ENABLED
+
+// Include custom WiFi API
+#include "wifi.h"
```

Comment:
wifi.h include flags.h, don;t need the flag.h include if i move this one up

Reply:
addressed in: https://github.com/jCallon/squirt/pull/1/commits/7c45fb1afcd46a0976bd7b50a193a305785b00db